kinde-j2ee-app/README.md

```diff
@@ -0,0 +1,59 @@
+# Kinde-J2EE-App
+
+This is an example J2EE servlet application designed to demonstrate how to implement authentication using Kinde. The application can be deployed on Apache Tomcat, a widely-used J2EE application server.
+
+## Pre-Setup
+
+Before you can run this authentication example, you'll need to register for an account on Kinde. This is required to configure the authentication settings for your application.
+
+### Steps to Register on Kinde:
```

Comment:
I think this section can be deleted - we really just need the link to the registration page as per above.